airgun/entities/computeresource.py

```diff
@@ -75,6 +76,17 @@ def vm_poweroff(self, rhev_name, vm_name):
         if vm['Power'].widget.read() == 'On':
             vm['Actions'].widget.click(handle_alert=True)
 
+    def vm_import(self, rhev_name, vm_name, hostgroup, location):
+        """Imports the specified VM"""
+        view = self.navigate_to(self, 'Detail', rhev_name=rhev_name)
+        view.virtual_machines.search(vm_name)
+        view.virtual_machines.more_button.click()
+        view.virtual_machines.import_button.click()
+        # we are already at import view
+        view = self.navigate_to(self, 'Import', rhev_name=rhev_name)
+        view.fill({'host.hostgroup': hostgroup, 'host.location': location})
```

Comment:
that is not responsibility of an entity layer. Always use:
`view.fill(values)`

Reply:
I am using an unnamed dictionary rather than putting it into a variable first. What's the difference?

Reply:
in your implementation you will be able to fill only two fields, in mine hundreds